app/models/display_ad.rb

```diff
@@ -37,10 +37,11 @@ class DisplayAd < ApplicationRecord
                              search: "%#{term}%"
                      }
 
-  def self.for_display(area, user_signed_in, article_tags = [])
+  def self.for_display(area, user_signed_in, organization_id = nil, article_tags = [])
     DisplayAds::FilteredAdsQuery.call(
       display_ads: self,
       area: area,
+      organization_id: organization_id || organization.id,
```

Comment:
@lboogie04 I'm maybe not following you here, so let me hop up a level:

* `FilteredAdsQuery` is fetching *all* matching `DisplayAd` records from the `display_ads` table
* Any `DisplayAd` with the `in-house` type always matches
* A `DisplayAd` with the `community` type might match, if the record's `organization_id` matches the `organization_id` on the currently-viewed Article

What I have been thinking we need from the callers, would be to pass in the currently viewed Article's organization ID, so that the query can find matching `display_ads` rows, by comparing the passed-in `organization_id` to the stored data.